ukrainian/dream/dream.md

```diff
@@ -0,0 +1 @@
+Ви бачите перед собою зелений пагорб з величезним дубом на вершині. К дубу мчить величезний білий кролик, який кожну мить поглядає на золотий кишеньковий годинник та щось перелякано бубонить собі під ніс. Він пригає до нори у самого коріння дуба. Ви поспішаєте та пригаєте йому вслід і починаєте падати - замість нори ви опиняєтеся у величезному колодязі, у якому навіть на бачите дна. Та [прокидаєтеся у холодному поту](../marshmallow.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.